tracker/logger.py

```diff
@@ -0,0 +1,20 @@
+"""
+This file contains the DailyLogFile observer that is used by twistd to save
+tracker logs.
+"""
+from twisted.python.log import FileLogObserver
+from twisted.python.logfile import DailyLogFile
+
+# TODO: Hard code the paths for now. Need to find a better solution.
```

Comment:
@whirm the `tracker_plugin.py` is a `MultiService`. How can I change it to an `Application`?